$OpenBSD: patch-posix-power_Makefile,v 1.1 2011/10/08 15:02:51 mpi Exp $
Needed to assemble tas.c that uses inline assembly "portable" to Mac OSX.
--- posix-power/Makefile.orig	Sat Oct  8 13:12:04 2011
+++ posix-power/Makefile	Sat Oct  8 13:20:17 2011
@@ -2,6 +2,8 @@
 include ../Make.config
 LIB=../libmachdep.a
 
+CFLAGS+= -Wa,-mregnames 
+
 OFILES=\
 	getcallerpc.$O\
 	md5block.$O\
